packages/esm-patient-labs-app/src/lab-orders/add-lab-order/lab-order-form.component.tsx

```diff
@@ -50,6 +46,7 @@ export function LabOrderForm({
   const { t } = useTranslation();
   const isTablet = useLayoutType() === 'tablet';
   const session = useSession();
+  const [isEditing, setIsEditing] = useState(initialOrder && initialOrder.action === 'REVISE');
```

Comment:
Minor observation: The setter seems redundant maybe we just memoize the value?

```jsx
const isEditing = useMemo(() => initialOrder && initialOrder.action === 'REVISE', [initialOrder]);

```

Reply:
+1